src/roiextractors/extractors/tiffimagingextractors/scanimagetiffimagingextractor.py

```diff
@@ -72,6 +75,10 @@ def __init__(
             file detection heuristics. Use this if automatic detection does not work correctly and you know
             exactly which files should be included.  The file paths should be provided in an order that
             reflects the temporal order of the frames in the dataset.
+        slice_sample : int, optional
```

Comment:
What if a user wants to include multiple frames per slice as in https://github.com/catalystneuro/neuroconv/issues/1260#issuecomment-2738055264?

Reply:
Not implemented yet. I need to think more carefully about the timing in that case and the resources for the Kim Conversion are not enough for me to implement at the moment.

Let me implement a draft so you get an idea of what would look though.

Reply:
Here you go:

https://github.com/catalystneuro/roiextractors/pull/420

Reply:
Thanks for the draft implementation, that was very helpful.

Based on the issues I flagged there, I think a bit more care is needed with frames_per_slice than just treating it as an artificial depth. Probably, the frame_per_slice_index needs to be another mapped entry in the ifd table.

Reply:
I think the PR you suggested is an improvement, and the readability and clarity are more important than generalization in this caso. I also think it's good that you took a closer look at the IFD mapping approach. Thanks for taking the time to review this. Feel free to merge your PR here.